### ISSUE TITLE
Documentaton for Alignment and Distribution Values

### ISSUE THREAD

**Issue by primehalo** [oarrabi/OAStackView#82]
Would it be possible to add documentation for what the Alignment and Distribution Values mean? The readme file notes the meaning of the Axis value (0 for Horizontal, 1 for Vertical) but doesn't specify anything about the Alignment or Distribution values.

**Comment by Darthpwner** [comment]
I agree, it is not helpful at all without this information. Also, iOS 9 Stack Views preserve alignment automatically when you drag new elements in. It appears this repo does not have this feature implemented.
